test/utilities/outputView.ts

```diff
@@ -58,68 +52,59 @@ export async function attemptToFindOutputPanelText(
 ): Promise<string | undefined> {
   await selectOutputChannel(outputChannelName);
 
-  while (attempts > 0) {
-    const outputViewText = await getOutputViewText();
-    if (outputViewText.includes(searchString)) {
-      return outputViewText;
+  browser.waitUntil(
+    async () => {
+      const outputViewText = await getOutputViewText();
+      return outputViewText.includes(searchString);
+    },
+    {
+      timeout: 1000 * attempts, // Convert attempts to milliseconds assuming each attempt is 1 second
+      interval: 500, // Check every second
```

Comment:
Why is the interval 500 instead of 1000?

Reply:
reduce overall wait time

Reply:
I'll fix the comment